apps/api/src/auth/session.serializer.ts

```
@@ -0,0 +1,19 @@
+import { Injectable } from '@nestjs/common';
+import { PassportSerializer } from '@nestjs/passport';
+import { UsersService } from '../users/users.service';
+
+@Injectable()
+export class SessionSerializer extends PassportSerializer {
+  constructor(private readonly usersService: UsersService) {
+    super();
+  }
+
+  serializeUser(user: any, done: Function) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)